app/assets/javascripts/discourse/controllers/preferences/users.js.es6

```diff
@@ -0,0 +1,18 @@
+import PreferencesTabController from "discourse/mixins/preferences-tab-controller";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+
+export default Ember.Controller.extend(PreferencesTabController, {
+  saveAttrNames: ["muted_usernames", "ignored_usernames"],
+
+  actions: {
+    save() {
+      this.set("saved", false);
+      return this.get("model")
+        .save(this.get("saveAttrNames"))
+        .then(() => {
```

Comment:
you can online this:

`.then(() => this.set("saved", true))`

Reply:
Cool, I think we need to change it in a couple other places under `preferences/*` too.

Reply:
It's ok if it's not in your PR, and it's not something very important, but if we can avoid adding more that's cool :p

Reply:
Let's avoid it ☺️